src/Controllers/ItemsController.cs

```diff
@@ -11,11 +12,11 @@ namespace Phytime.Controllers
     [Route("api/[controller]")]
     public class ItemsController : Controller
     {
-        private readonly PhytimeContext _context;
-        private const string SessionListItemsKey = "items";
-        public ItemsController(PhytimeContext context)
+        private readonly IRepository _repository;
+
+        public ItemsController(IConfiguration config, IRepository repository = null)
```

Comment:
You shouldn't you IConfigureation here, you should use IOption<ApplicationOption> for example
You can refer it from here https://github.com/ArtemGontar/Quiz/blob/1617b534172a49e5b240bdbe3bf6bc7bc2518c77/AppQuiz.Api/Startup.cs#L41
or
https://metanit.com/sharp/aspnet5/6.3.php